requirements.txt

```diff
@@ -1,14 +1,10 @@
-Django==1.7.1
-Markdown==2.1.1
-PyYAML==3.11
-argh==0.23.2
-argparse==1.2.1
-coverage==3.6
-django-nose==1.2
-djangorestframework>=2.3.8
-django-filter==0.11.0
-jsonschema==2.5
-nose==1.3.0
-mock==1.0.1
-ordereddict==1.1
-docutils>=0.11
\ No newline at end of file
+Django==1.9.7
+djangorestframework==3.4.0
+coreapi==1.32.0
+openapi-codec==1.0.0
+tox==2.3.1
+Pygments==2.0.2
+Markdown==2.6.3
+simplejson==3.8.2
```

Comment:
Really curious, is there any reason why you're requiring `simplejson` over the built-in `json` module?